lightning/src/ln/monitor_tests.rs

```diff
@@ -82,6 +83,17 @@ fn chanmon_fail_from_stale_commitment() {
 	expect_payment_failed_with_update!(nodes[0], payment_hash, false, update_a.contents.short_channel_id, true);
 }
 
+fn test_spendable_output<'a, 'b, 'c, 'd>(node: &'a Node<'b, 'c, 'd>, spendable_tx: &Transaction) {
+	let mut spendable = node.chain_monitor.chain_monitor.get_and_clear_pending_events();
+	assert_eq!(spendable.len(), 1);
+	if let Event::SpendableOutputs { outputs } = spendable.pop().unwrap() {
```

Comment:
Seems like we'd want to assert that we actually get a `Event::SpendableOutputs` -- this will currently pass if it's not.